tests/test_player.py

```diff
@@ -0,0 +1,20 @@
+#Import libraries
```

Comment:
All the comments in this file seems unnecessary because we know it's a test file and the variable names and assignments are very clear.